telemetry/convert.py

```diff
@@ -36,6 +37,8 @@ class Converter:
     """A class for converting incoming payloads to a more compact form"""
     VERSION_UNCONVERTED = 1
     VERSION_CONVERTED = 2
+    # the unified FHR/Telemetry ping has a top-level version of 2 and payload version of 4
+    VERSION_UNIFIED = 2
```

Comment:
I filed bug 1135783 to have version=4 in unified pings (incl saved-session). My script will copy version to ver, so ver will also be 4.
Should I add another version number to represent unified saved-session pings that were converted to look like classic pings?